letta/schemas/agent.py

```diff
@@ -107,6 +107,16 @@ class CreateAgent(BaseModel, validate_assignment=True):  #
     include_base_tools: bool = Field(True, description="The LLM configuration used by the agent.")
     description: Optional[str] = Field(None, description="The description of the agent.")
     metadata_: Optional[Dict] = Field(None, description="The metadata of the agent.", alias="metadata_")
+    llm: Optional[str] = Field(
+        None,
+        description="The LLM configuration handle used by the agent, specified in the format "
+        "provider/model-name, as an alternative to specifying llm_config. This field can also "
+        "be used to override the context window by optionally appending ':context_window'.",
+    )
+    embedding: Optional[str] = Field(
+        None, description="The embedding configuration handle used by the agent, specified in the format provider/model-name."
+    )
+    context_window: Optional[int] = Field(None, description="The context window specification used by the agent.")
```

Comment:
could we rename this `context_window_limit` and also add an `embedding_chunk_size` field? For `embedding_chunk_size` we might want to add a constant somewhere that defines the default